Drop environment marker in extras_require. Some older setuptools don't support
that, and this is not used anyway within robotpkg

--- setup.py~	2022-05-27 21:31:45.000000000 +0200
+++ setup.py	2023-01-13 11:45:30.378234121 +0100
@@ -52,7 +52,6 @@
             'flake8-docstrings',
             'flake8-import-order',
             'flake8-quotes',
-            "mock; python_version < '3.3'",
             'pytest',
         ]},
 }
